### PR TITLE
Add Code of Conduct.

### PR DESCRIPTION
**Pull request by octo**
Add the "Contributor Covenant" as collectd's code of conduct, as discussed at the collectd meetup in Munich.

ChangeLog: collectd: A Code of Conduct has been added.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+**Community Impact**: Demonstrating a pattern of violation of community
+standards, including sustained inappropriate behavior,  harassment of an
+individual, or aggression toward or disparagement of classes of individuals.
+
+**Consequence**: A permanent ban from any sort of public interaction within
+the community.
+
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor
```
Comment (jkohen) [inline CODE_OF_CONDUCT.md]: Could you summarize the changes? I'm curious about the different needs of this community.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement. Currently
+responsible are:
+
+*   Florian Forster
```
Comment (mrunge) [inline CODE_OF_CONDUCT.md]: can we please add email addresses to reach these?

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (octo) [reply CODE_OF_CONDUCT.md]: I split this into two commits: a verbatim import of the *Contributor Covenant* and the local changes.

To summarize: the contact method has been added and the initial heading has been slightly altered.

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (octo) [reply CODE_OF_CONDUCT.md]: Done.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (jkohen) [reply CODE_OF_CONDUCT.md]: Thanks!
